NEXTROLL_CHANGES.md

```diff
@@ -366,6 +366,15 @@ aws ec2 create-default-vpc --region us-west-2
 
 The artifacts should build successfully after this.
 
+Go to AWS Console > AMIs and search for the AMI named  aggregation-service-enclave... Add the following tags:
```

Comment:
#nit
```suggestion
Go to AWS Console > AMIs and search for the AMI named aggregation-service-enclave... Add the following tags:
```

Reply:
thanks @edmilson-fnk - i fixed the spacing issue.